test/non-interference/http-e2e.js

```diff
@@ -73,7 +73,7 @@ glob(test_glob, function(err, files) {
           ' >' +  files[testCount] + '.instru.js' + '&& mv ' + files[testCount] +
           '.instru.js' + ' ' + files[testCount]);
     }
-    var results = cp.spawnSync('node', [files[testCount]]);
+    var results = cp.spawnSync('node', ['--allow_natives_syntax', files[testCount]]);
```

Comment:
@matthewloring Should the commit comments mention anything about this flag and what it does? I can't find any information about it just from a preliminary google search